packages/api/utils/validation.ts

```diff
@@ -25,9 +26,25 @@ export function isValidJson(str: string): boolean {
  * @returns true if the object has all the attributes of a {@link PortfolioStep}
  */
 export function isPortfolioStep(object: unknown): object is PortfolioStep {
-  // Ensure that the given item is a valid object prior to checks its members
+  // Ensure that the given item is a valid object prior to checking its members
   if (typeof object !== "object" || object === null) {
     return false;
   }
   return ["name", "description", "dod_components", "portfolio_managers"].every((item) => item in object);
 }
+
+/**
+ * Check whether a given object is a {@link FundingStep}.
+ *
+ * Note that this only asserts that the given object meets the interface. It does not validate
+ * that the object is a valid {@link FundingStep}.
+ *
+ * @param object - The object to check
+ * @returns true if the object has all the attributes of a {@link FundingStep}
+ */
+export function isFundingStep(object: unknown): object is FundingStep {
+  if (typeof object !== "object" || object === null) {
+    return false;
+  }
+  return ["task_order_number", "task_order_file", "csp", "clins"].every((item) => item in object);
```

Comment:
Good for now! (In another ticket) we'll need to add support for more advanced validation than just "does the field exist".